test/powershell/Modules/Microsoft.PowerShell.Management/Get-PSProvider.Tests.ps1

```diff
@@ -24,4 +24,56 @@ Describe "Get-PSProvider" -Tags "CI" {
 
 	{ $actual | Format-List } | Should -Not -Throw
     }
+
+    Context 'PathSeparators' {
+        BeforeAll {
+            $windowsTestCases = @(
+                @{Provider = 'FileSystem'; Value = @("\", "/")}
+                @{Provider = 'Variable'; Value = @("\", "/")}
+                @{Provider = 'Function'; Value = @("\", "/")}
+                @{Provider = 'Registry'; Value = @("\")}
+            )
+
+            $nonWindowsTestCases = @(
+                @{Provider = 'FileSystem'; Value = @("/", "\")}
+                @{Provider = 'Variable'; Value = @("/", "\")}
+                @{Provider = 'Function'; Value = @("/", "\")}
+            )
+
+            $testCases = if ($IsWindows) {
+                            $windowsTestCases
+                        }
+                        else {
+                            $nonWindowsTestCases
+                        }
```

Comment:
Not blocking. It's better to change to the following, so we can avoid unnecessary code on a specific platform.
```
$testCases = if ($IsWindows) {
    @{Provider = 'FileSystem'; Value = @("\", "/")}
    ...
} else {
    @{Provider = 'FileSystem'; Value = @("/", "\")}
    ...
}
```

Reply:
How is that different to what I am currently doing?

Reply:
Dongbo mean that currently we create extra variable $windowsTestCases or  $nonWindowsTestCases  - we need one not both.

Reply:
Done

Reply:
In your original code, both `$windowsTestCases` and `$nonWindowsTestCases` will be created and initialized no matter what platform the test is running on.